azuredeploy.json

```diff
@@ -24,12 +24,68 @@
             "type": "bool"
         },
         "azureBackupSwitch": {
-            "defaultValue": false,
+            "defaultValue": true,
             "metadata": {
                 "description": "Switch to configure AzureBackup and enlist VM's"
             },
             "type": "bool"
         },
+        "azureSearchSwitch": {
+            "defaultValue": false,
```

Comment:
This is strange. What if both "installElasticSearchSwitch" and "azureSearchSwitch" are true? Will we have both of them? Is that what we wanted?

I thought we agreed to have a 3-way switch for the Moodle global search option: None, ElasticSearch, AzureSearch. Any reason why this was added separately, not replacing the existing "installElasticSwitch" param with a 3-way switch?

If it was simply because of the syntactic limitation you might have thought, there are many other Azure Resource Manager template functions that will support a 3-way switch and achieve the same, so please do refer the [documentation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-functions) and update this.